### PR TITLE
SocketStore Service Tests

### PR DESCRIPTION
**Pull request by Dire-0x**
Created a test dir. We should write all service tests in test/unit/services. Created a lib folder in test/lib where we can keep mocked instances. The SocketStore Service didn't need any mocked instances however services like the RedisService will need a mocked Redis instance. I'll move to the RedisService tests next so you can see how to create the mocks.

Includes a few tests.

- Should properly get, set, & unset a sockets socketInfo
- Should property get active pairs
- Should properly get sockets for pair
- Should properly move socket to another active pair

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   coverageDirectory: 'coverage',
   preset: 'ts-jest',
   testEnvironment: 'node',
+  testMatch: ['**/?(*.)+(spec|test).ts?(x)'],
```
Comment (henridf) [inline jest.config.js:9]:
```suggestion
  testMatch: ['**/?(*.)+test).ts?(x)'],
```

File: package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
     "watch": "tsc -w",
     "dev": "cross-env NODE_ENV=development nodemon dist/index.js",
     "start": "cross-env NODE_ENV=production node dist/index.js",
-    "test": "cross-env NODE_ENV=test jest src",
-    "test:watch": "cross-env NODE_ENV=test jest src--watch",
+    "test": "cross-env NODE_ENV=test jest test --verbose",
+    "test:watch": "cross-env NODE_ENV=test jest test --watch --verbose",
     "build": "tsc",
     "lint": "eslint src --fix",
```
Comment (henridf) [inline package.json:13]:
```suggestion
    "lint": "eslint src test --fix",
```

**Comment by henridf** [comment]
🥳 for the first test!

File: tsconfig.json [hunk 3]
```diff
@@ -33,6 +33,6 @@
       "@constants/*": ["constants/*"]
     }
   },
-  "exclude": ["node_modules"],
-  "include": ["./src/**/*.tsx", "./src/**/*.ts"]
+  "exclude": ["node_modules", "test"],
```
Comment (henridf) [inline tsconfig.json:36]: why is test being added to both exclude and include here?

Comment (Dire-0x) [reply tsconfig.json:36]: Well its kind of a hack. If I don't include test then all the imports using the '@services/...' syntax break, however we want to exclude it from dist.

Comment (henridf) [reply tsconfig.json:36]: ah kk

**Review by henridf — APPROVED** [review]
👍  on green build. (looks like my regex suggested change broke something)

**Review by donvincenz0 — APPROVED** [review]

**Pull request merged**